app/controllers/insights_cloud/hits_controller.rb

```diff
@@ -0,0 +1,17 @@
+module InsightsCloud
+  class HitsController < ::ApplicationController
+    def index
+      host = Host.where(id: hits_params).first
+
+      render json: {
+        hits: host.insights.hits,
+      }, status: :ok
+    end
+
+    private
+
+    def hits_params
```

Comment:
```suggestion
    def host_id_param
```
Just to be a bit more clear with the value.